src/types.h

```diff
@@ -18,6 +18,7 @@
 #include <sstream>                          // For ostringstream
 #include <mutex>                            // For mutex, unique_lock
 #include <memory>                           // For unique_ptr
+#include <iostream>
```

Comment:
why is this required?

Reply:
The easylogging++.h brings this header inclusion into the build and some of the other librealsense files fail compilation otherwise.